c/httpserver.c

```diff
@@ -62,6 +62,7 @@
 #include "charsets.h"
 #include "impersonation.h"
 #include "httpserver.h"
+#include "zccLogging.h"
```

Comment:
I would avoid using official message IDs in this file. Since this is common code, adding message IDs makes it impossible to reuse in other projects. Perhaps we need to provide hooks, or some message ID parameter when creating the http server in `zss.c`, so it could prepend its messages with the message ID.

Alternatively, all message would be debug. If we start having issues, we enable tracing.